context.c

```diff
@@ -392,6 +392,30 @@ int iio_context_get_version(const struct iio_context *ctx,
 	return 0;
 }
 
+unsigned int iio_context_get_version_major(const struct iio_context *ctx)
+{
+	if (ctx && ctx->git_tag)
+		return ctx->major;
```

Comment:
just curious: Do we only get major/minor if there's a context git_tag? Looks a bit odd to me (just by looking at this code and variable names).

Reply:
Yes, basically if `ctx->git_tag` is initialized, then it was created with the XML backend, and it is safe to assume that ctx->major and ctx->minor have been initialized as well. Otherwise you're either running with the local backend or a very old version of IIOD, and in this case you return the library version.